src/documents/ModelRelay.md

```diff
@@ -67,21 +102,21 @@ This style is used when we require a [Model Transformation](to-do/ModelTransform
     </thead>
     <tbody>
         <tr>
-            <td>source</td>
+            <td><code>source</code> (usual, but not required)</td>
             <td>String (EL Reference or short local path)</td>
-            <td>The source model path to be linked by this relay rule</td>
+            <td>The source model path to be linked by this relay rule. This option can be omitted if all arguments to the transform are presented as IoC references in its definition &#8212; as, for example, with `fluid.transforms.free`.</td>
             <td><code>"volume"</code> / <code>"{someComponent}.model.volume"</code></td>
         </tr>
         <tr>
-            <td>target</td>
+            <td><code>target</code> (required)</td>
             <td>String (EL Reference or short local path)</td>
-            <td>The target model path to be linked by this relay rule</td>
+            <td>The target model path to be linked by this relay rule. This option must always be present.</td>
```

Comment:
Do we need to add "This option must always be present" when we've already added "(required)"?